README.md

```diff
@@ -148,6 +148,16 @@ For messages permanently failed in the dead letter queue (XQ), query:
 HGETALL dramatiq:default.XQ.msgs
 ```
 
+To query a list of missing blocks in a range, run:
+```
+psql postgresql://postgres:password@localhost:5432/mev_inspect -c "select generate_series(13999471, 13999571) EXCEPT (select distinct block_number from blocks)" --csv -t
```

Comment:
I think ok to leave it at purely the functionality to backfill a list of blocks since that could come from many sources

ex:
```suggestion

#### Backfilling a list of blocks

Create a file containing a block per row, for example `blocks.txt` containing:
```
12500000
12500001
12500002
```

Then queue the blocks with 
``` 
cat blocks.txt | ./mev block-list
```
```

Reply:
How about now?